DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -1011,6 +1011,10 @@ globus_l_gfs_iRODS_destroy(
 
         iRODS_disconnect(iRODS_handle->conn, "main_thread");
 
+        if (iRODS_handle->conn) {
+            cleanRcComm(iRODS_handle->conn);
```

Comment:
This _cleans_ the RcComm.

My guess is that only frees internal memory. Does `conn` need to be free'd too or is it not a pointer?

Reply:
It is a pointer.  I initially added in a free for it but then the forked process never stopped and I think it was having some type of problem in globus_free(iRODS_handle).  I'll look more into it.

Reply:
We discovered that rcDisconnect() cleans this up so there is no need to call cleanRcComm() or free(conn).